src/app/feature/theme/services/theme.service.ts

```diff
@@ -21,12 +21,12 @@ export class ThemeService {
   }
 
   init() {
-    // Retrieve the last active theme with default fallback
+    // Retrieve the last active theme and apply it, with default fallback
     const currentThemeName = this.getCurrentTheme() ?? this.defaultThemeName;
-    this.setTheme(currentThemeName);
+    this.setTheme(currentThemeName, true);
   }
 
-  public setTheme(themeName: string) {
+  public setTheme(themeName: string, isInit = false) {
```

Comment:
nit(non-blocking)
I'm not quite sure what the `isInit` parameter does. Could you rename it to something more explicit or add an `@param` comment? Or perhaps it might make more sense to include the logic in the init method itself? E.g. attempt to set the theme, and if not successful set default theme during init (assuming we have a `getTheme` or similar that can be used to check if the theme has been set)